### DISCUSSION TITLE
About directions and its identification

### DISCUSSION THREAD

**Discussion by jacobosaa** [Anura3D/Anura3D_OpenSource#180] [General]
Hi everybody! I'm trying to analize the shear stress during a column collapse model, but I can't find the correct way to identify it. I only know that it must be on Eff_stress_solid and  horizontal stress is (4) and vertical is (0) so maybe the shear stress is in another direction between them. Let me add that my model is 2D. 
what are the other directions? (1,2,3,5,6,7,8) ?

**Comment by WaveHello** [comment]
Hello jacobosaa,

Please see the figure below for the map between the stress tensor and the paraview numbering. The figure came from the 11th page of the Anura3D tutorial manual.

![image](https://github.com/Anura3D/Anura3D_OpenSource/assets/110841896/059b2707-2b9e-475d-a8b7-ed87ec62304c)
